examples/nextjs/app/ui/dashboard/session-details.tsx

```diff
@@ -0,0 +1,19 @@
+import { jwtDecode } from 'jwt-decode';
+import { cookies } from 'next/headers';
+
+export default function SessionDetails() {
+  const cookieStore = cookies();
```

Comment:
When I access this page directly without being logged in I get this error
![image](https://github.com/corbado/javascript/assets/134682658/496b78b5-befc-43ff-8c3d-a2c4bf34b0a5)

Other pages might be affected as well, I haven't checked.